underscore.js

```diff
@@ -1446,6 +1453,13 @@
     return _.isFunction(value) ? value.call(object) : value;
   };
 
+  // Returns the, possibly deeply nested, value at `path` if defined.
+  // Otherwise, returns `fallback`.
+  _.get = function(object, path, fallback) {
+    var value = property(path)(object);
+    return value === void 0 ? fallback : value;
+  };
```

Comment:
I missed that in the commit message since I don't usually see the commit messages directly. We typically leave commit messages short (~1 sentence) and then write longer prose in the body of the Pull Request. Mind either copying that message to the pull request, or pointing out that there might be more text in the commit message in the future?